websites/H/HBO Max/presence.ts

```diff
@@ -44,13 +44,32 @@ const presence = new Presence({
 		"sesame-workshop": "Sesame Workshop",
 		"looney-tunes": "Looney Tunes",
 		crunchyroll: "Crunchyroll Collection"
-	};
+	},
+	coverUrls: Record<string, string> = {};
+
+function fetchCover(): Promise<string> {
+	return new Promise(resolve => {
+		fetch(
+			`https://comet.api.hbo.com/express-content/${
+				location.pathname.split("/")[2]
+			}?device-code=desktop&product-code=hboMax&api-version=v9.0&country-code=US`
+		)
+			.then(x => x.json())
+			.then(x =>
+				resolve(
+					`https://artist.api.cdn.hbo.com/images/${
+						x[0].body.references.series.match(/series:([^:]+)/)[1]
+					}/tileburnedin?size=1024x1024`
+				)
+			);
+	});
+}
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
 			largeImageKey: "lg"
 		},
-		video: HTMLVideoElement = document.querySelector("video"),
+		video = document.querySelector("video"),
```

Comment:
Why did you remove the type?
```suggestion
		video = document.querySelector<HTMLVideoElement>("video"),
```

Reply:
Because it's unnecessary, this query already gave you a type...